lib/jobs/general-redfish-catalog.js

```diff
@@ -162,9 +171,32 @@ function GeneralRedfishCatalogJobFactory(
             return waterline.catalogs.count(query)
             .then(function(count) {
                 if (count) {
-                    return waterline.catalogs.update(query, dataToSave);
+                    return waterline.catalogs.find(query)
+                    .then(function(catalog) {
+                        if (!_.isEqual(catalog[0].data, dataToSave.data)) {
+                            return waterline.catalogs.update(query, dataToSave);
+                        }
+                    });
                 } else {
-                    return waterline.catalogs.create(dataToSave);
+                    return waterline.catalogs.create(dataToSave)
+                    .then(function() {
+                        return waterline.catalogs.update(query, dataToSave);
```

Comment:
We are doing the update to get the "."'s back in the odata keys.
This fixes the entries back to what is expected for redfish data.